CRITICAL - 2020-12-19 00:37:23 --> Unable to connect to the database.
#0 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp7\htdocs\apps\sidamis\app\Controllers\Login.php(16): CodeIgniter\Database\BaseConnection->query('select * from p...')
#2 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Login->auth()
#3 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#4 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp7\htdocs\apps\sidamis\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-12-19 00:37:42 --> Undefined index: icon
#0 C:\xampp7\htdocs\apps\sidamis\app\Libraries\Navigation.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp7\\htdoc...', 32, Array)
#1 C:\xampp7\htdocs\apps\sidamis\app\Libraries\Navigation.php(15): App\Libraries\Navigation->gen_menu()
#2 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\View\Cell.php(149): App\Libraries\Navigation->menu()
#3 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Common.php(1140): CodeIgniter\View\Cell->render('\\App\\Libraries\\...', NULL, 0, 'AppLibrariesNav...')
#4 C:\xampp7\htdocs\apps\sidamis\app\Views\template\default.php(110): view_cell('\\App\\Libraries\\...')
#5 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\View\View.php(246): include('C:\\xampp7\\htdoc...')
#6 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\View\View.php(257): CodeIgniter\View\View->render('template/defaul...', Array, true)
#7 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('global/list', Array, true)
#8 C:\xampp7\htdocs\apps\sidamis\app\Controllers\Admin\Home.php(17): view('global/list', Array)
#9 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Home->index()
#10 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Home))
#11 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp7\htdocs\apps\sidamis\public\index.php(45): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2020-12-19 00:39:56 --> pg_query(): Query failed: ERROR:  relation &quot;informasi&quot; does not exist
LINE 2: FROM &quot;informasi&quot;
             ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp7\\htdoc...', 196, Array)
#1 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\Postgre\Connection.php(196): pg_query(Resource id #184, 'SELECT *\nFROM "...')
#2 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 C:\xampp7\htdocs\apps\sidamis\app\Models\Informasi_model.php(12): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp7\htdocs\apps\sidamis\app\Controllers\Informasi.php(21): App\Models\Informasi_model->get()
#7 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\informasi->index()
#8 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\informasi))
#9 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp7\htdocs\apps\sidamis\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-19 00:40:19 --> pg_query(): Query failed: ERROR:  relation &quot;gallery&quot; does not exist
LINE 2: FROM &quot;gallery&quot;
             ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp7\\htdoc...', 196, Array)
#1 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\Postgre\Connection.php(196): pg_query(Resource id #184, 'SELECT *\nFROM "...')
#2 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 C:\xampp7\htdocs\apps\sidamis\app\Models\Gallery_model.php(13): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp7\htdocs\apps\sidamis\app\Controllers\Gallery.php(21): App\Models\Gallery_model->get()
#7 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Gallery->index()
#8 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Gallery))
#9 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp7\htdocs\apps\sidamis\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-19 01:04:46 --> pg_query(): Query failed: ERROR:  syntax error at or near &quot;d033e22ae348aeb5660fc2140aec35850c4da997&quot;
LINE 1: ...er_name='admin\'#' and public.user.user_password='d033e22ae3...
                                                             ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp7\\htdoc...', 196, Array)
#1 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\Postgre\Connection.php(196): pg_query(Resource id #176, 'select * from p...')
#2 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\Postgre\Connection->execute('select * from p...')
#3 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select * from p...')
#4 C:\xampp7\htdocs\apps\sidamis\app\Controllers\Login.php(16): CodeIgniter\Database\BaseConnection->query('select * from p...')
#5 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Login->auth()
#6 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp7\htdocs\apps\sidamis\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-19 21:01:05 --> pg_query(): Query failed: ERROR:  column &quot;kar_visible&quot; does not exist
LINE 1: select *, art_id as id from art where kar_visible is true or...
                                              ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp7\\htdoc...', 196, Array)
#1 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\Postgre\Connection.php(196): pg_query(Resource id #181, 'select *, art_i...')
#2 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\Postgre\Connection->execute('select *, art_i...')
#3 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('select *, art_i...')
#4 C:\xampp7\htdocs\apps\sidamis\app\Libraries\SmartComponent\Datasource.php(73): CodeIgniter\Database\BaseConnection->query('select *, art_i...')
#5 C:\xampp7\htdocs\apps\sidamis\app\Libraries\SmartComponent\Grid.php(172): App\Libraries\SmartComponent\Datasource->run()
#6 C:\xampp7\htdocs\apps\sidamis\app\Libraries\SmartComponent\Grid.php(127): App\Libraries\SmartComponent\Grid->get_datasource()
#7 C:\xampp7\htdocs\apps\sidamis\app\Controllers\Admin\Coba.php(49): App\Libraries\SmartComponent\Grid->output()
#8 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Coba->grid()
#9 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Coba))
#10 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp7\htdocs\apps\sidamis\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2020-12-19 21:01:05 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Format\Exceptions\FormatException.php:9
Stack trace:
#0 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Format\JSONFormatter.php(71): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\API\ResponseTrait.php(414): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\API\ResponseTrait.php(134): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp7\htdocs\apps\sidamis\vendor\codeigniter4\framework\system\Debug\Exceptions.php(168): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
